python/python-spf-engine/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
Please apply the above comments to this port.

Reply:
is this a Python program or (also) intended to be imported as a module. From a quick look it appears to be the first; if so, the port should probably not start with “py-“ nor be in the main “python” category.

Reply:
so, what is this port: a program or a module?